lib/browser/navigation.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+var Url = require('url');
+
+var _ = require('lodash');
+var debug = require('debug')('testium-driver-sync:navigation');
+
+var makeUrlRegExp = require('./makeUrlRegExp');
+var waitFor = require('./wait');
+
+exports._forwarded = [
+  'refresh',
+  'getUrl',
```

Comment:
I was looking for a well-documented style guide and remembered seeing the airbnb one. I didn't spend too much time choosing it to be honest. It seemed reasonably close to what we were doing anyhow.